components/nexus-common/src/main/java/org/sonatype/nexus/common/time/DateHelper.java

```diff
@@ -1,6 +1,6 @@
 /*
  * Sonatype Nexus (TM) Open Source Version
- * Copyright (c) 2008-present Sonatype, Inc.
+ * Copyright (c) 2008-2015 Sonatype, Inc.
```

Comment:
Remember to rebase/merge with latest master and re-generate the headers with `mvn license:format -N`